src/python/install.sh

```diff
@@ -7,24 +7,24 @@
 # Docs: https://github.com/microsoft/vscode-dev-containers/blob/main/script-library/docs/python.md
 # Maintainer: The VS Code and Codespaces Teams
 
-PYTHON_VERSION=${VERSION:-"latest"} # 'system' checks the base image first, else installs 'latest'
-INSTALL_PYTHON_TOOLS=${INSTALLTOOLS:-"true"}
-OPTIMIZE_BUILD_FROM_SOURCE=${OPTIMIZE:-"false"}
-PYTHON_INSTALL_PATH=${INSTALLPATH:-"/usr/local/python"}
-OVERRIDE_DEFAULT_VERSION=${OVERRIDEDEFAULTVERSION:-"true"}
+PYTHON_VERSION="${VERSION:-"latest"}" # 'system' or 'os-provided' checks the base image first, else installs 'latest'
+INSTALL_PYTHON_TOOLS="${INSTALLTOOLS:-"true"}"
+OPTIMIZE_BUILD_FROM_SOURCE="${OPTIMIZE:-"false"}
```

Comment:
```suggestion
OPTIMIZE_BUILD_FROM_SOURCE="${OPTIMIZE:-"false"}"
```

I wonder if the Python test failures are due to missing closing quote.